lib/src/ui.dart

```diff
@@ -10,29 +10,107 @@ import 'network_event.dart';
 import 'network_logger.dart';
 
 /// Overlay for [NetworkLoggerButton].
-class NetworkLoggerOverlay extends StatelessWidget {
-  NetworkLoggerOverlay._({Key? key}) : super(key: key);
+class NetworkLoggerOverlay extends StatefulWidget {
+  NetworkLoggerOverlay._({this.right, this.bottom, Key? key}) : super(key: key);
+
+  double? bottom;
```

Comment:
Not important but maybe we can also introduce `draggable` option with default value set `true` - can be used to disable dragging behavior.